File: 20MCA133/php/armstrong.php
<?php
if(isset($_POST["sub"]))
{
$a=$_POST["num1"];
$temp=$a;
$s=0;
while($a>0)
{
    $r = $a % 10;
    $c = $r*$r*$r;
    $s = $s+$c;
    $a = $a/10;
}
if($temp == $s)
    echo $temp." is armstrong";
else
    echo $temp." is not Armstrong";
}
?>
<form method="POST">
Enter number:<input type="text" name="num1">
<input type="submit" name="sub">
</form>

File: 20MCA131 - Programming/Armstrong.py
"""Program to check the number is Armstrong or not"""

#Enter input
num = int(input("Enter 3-digit number : "))

sum = 0
temp = num

#using loop
while temp > 0:
    digit = temp % 10
    sum = sum+(digit * digit * digit)
    temp = temp//10

if sum==num:
    print(num, 'is an Armstrong number')
else:
    print(num, 'is not an Armstrong number')
